src/buildercore/terraform.py

```diff
@@ -216,6 +217,7 @@ def render_fastly(context):
         # main
         linked_main_vcl = fastly.MAIN_VCL_TEMPLATE
         inclusions = [fastly.VCL_SNIPPETS[name].as_inclusion() for name in vcl_constant_snippets] + list(vcl_templated_snippets.values())
+        inclusions.reverse()
```

Comment:
why?

Reply:
Their position is determined by `#FASTLY error` or whatever, later entries appear first. Need to make sure that entries like https://github.com/elifesciences/builder/blob/fb476d4a9044640e87b3f15d9de394bd65003280/src/buildercore/terraform.py#L308 appear after https://github.com/elifesciences/builder/blob/fb476d4a9044640e87b3f15d9de394bd65003280/src/buildercore/terraform.py#L295.

Reply:
https://github.com/elifesciences/builder/blob/master/src/buildercore/fastly.py#L26 always inserts the lines from the snippet in the same place; hence snippets inserted earlier get pushed down after the ones inserted later. Reversing is clunky but works.